README.md

```diff
@@ -55,6 +55,7 @@ module "api_gateway" {
 | name                                    | The name of the created API Gateway                               |
 | id                                     | The id of the created API Gateway                               |
 | address                                 | The address of the deployed API Gateway      |
```

Comment:
Address is not an actual output in outputs.tf. I didn't address that as part of this PR.